src/index.tsx

```diff
@@ -197,6 +197,10 @@ export interface ReactTagsWrapperProps {
    * Whether to display a button to clear all the tags.
    */
   clearAll?: boolean;
+  /**
+   * accessibility la
```

Comment:
the JSDoc is incomplete

Reply:
my bad , I thought jsdoc is added looking at auto-complete, will add it

Reply:
added now https://github.com/react-tags/react-tags/blob/f07fc68cfbba2011073e9b33bcf2f5802b8518a8/src/index.tsx#L201